### DISCUSSION TITLE
Anyway to fill a dot line into a solid line?

### DISCUSSION THREAD

**Discussion by tonyfkchu** [dlemstra/Magick.NET#752] [Help]
Dear Expert, 

Any way to fill the dot line into a solid line? and the attached is the image that contain dot line

Best Regards
Tony Chu

![DotLine](https://user-images.githubusercontent.com/6896544/95308325-f6efa200-08b3-11eb-8fa5-f7d30c9fb9d9.PNG)

**Comment by dlemstra** [comment]
Do you have any ideas how we could do that @fmw42?
